src/components/LoginForm.tsx

```diff
@@ -0,0 +1,74 @@
+
+import * as React from 'react';
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import CustomText from './custom-components/CustomText';
+import LoginInput from './LoginInput';
+import ProviderAuthBlock from './ProviderAuthBlock';
+
+
+const loginForm = () => {
+  return (
+    <KeyboardAwareScrollView>
+      <View style={styles.loginForm}>
+        <CustomText style={styles.loginWelcomeText}>HI THERE 👋🏻 </CustomText>
+        <CustomText style={styles.title}>Let’s Get you Started</CustomText>
+        <LoginInput
+          placeholder={'Email'}
+          source={require('../../assets/images/icon_email.png')}
+        />
+        <LoginInput
+          placeholder={'Password'}
+          secureTextEntry={true}
+          source={require('../../assets/images/icon_password.png')} />
```

Comment:
Especially me, don’t like strict string path.
If you have a lot of images (and you have) create `index.ts` file in assets with imports like:
```

import passwordIcon from '../../assets/images/icon_password.png'
import emailIcon from '../../assets/images/icon_email.png'
...

export {
passwordIcon,
emailIcon
...
}
```